site/guide/6-state.md

```diff
@@ -177,18 +177,19 @@ three simple steps:
 
 Presently, Rocket provides built-in support for the following databases:
 
-| Kind     | Driver                | `Poolable` Type                | Feature                |
-|----------|-----------------------|--------------------------------|------------------------|
-| MySQL    | [Diesel]              | [`diesel::MysqlConnection`]    | `diesel_mysql_pool`    |
-| MySQL    | [`rust-mysql-simple`] | [`mysql::conn`]                | `mysql_pool`           |
-| Postgres | [Diesel]              | [`diesel::PgConnection`]       | `diesel_postgres_pool` |
-| Postgres | [Rust-Postgres]       | [`postgres::Connection`]       | `postgres_pool`        |
-| Sqlite   | [Diesel]              | [`diesel::SqliteConnection`]   | `diesel_sqlite_pool`   |
-| Sqlite   | [`Rustqlite`]         | [`rusqlite::Connection`]       | `sqlite_pool`          |
-| Neo4j    | [`rusted_cypher`]     | [`rusted_cypher::GraphClient`] | `cypher_pool`          |
-| Redis    | [`redis-rs`]          | [`redis::Connection`]          | `redis_pool`           |
-| MongoDB  | [`mongodb`]           | [`mongodb::db::Database`]      | `mongodb_pool`         |
-| Memcache | [`memcache`]          | [`memcache::Client`]           | `memcache_pool`        |
+<!-- Note: Keep this table in sync with contrib/lib/src/databases.rs -->
```

Comment:
Is there a better kind of comment to use here?

Reply:
I don't think so! This is totally good.